db/migrate/20230526133904_create_neighbour_letters.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreateNeighbourLetters < ActiveRecord::Migration[7.0]
+  def change
+    create_table :neighbour_letters do |t|
+      t.references :planning_application, null: false
+      t.string :address
+      t.string :text
+      t.string :sent_at
+      t.json :response
```

Comment:
That's a good idea, it needs to be associated with _something_, but perhaps the consultation is more useful.

I suppose it won't have more than one consultation for a given planning application, so this will be unambiguous either way? But referencing the consultation might at least be more convenient.

Reply:
Oh, actually, hang on, you've got a `neighbours` table, so presumably I can just reference that instead.

Reply:
Ah yes forgot the code that I wrote 4 days ago 😂